reciprocalspaceship/stats/completeness.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import pandas as pd
+import reciprocalspaceship as rs
+
+def compute_multiplicity(hobs, cell, spacegroup, anomalous=False, dmin=None):
```

Comment:
Let's change this to `compute_redundancy()` to avoid confusion with `DataSet.compute_redundancy()`